cbmgmtx/mgmt.go

```diff
@@ -583,7 +579,8 @@ func (h Management) decodeMutableBucketSettings(data *bucketSettingsJson) (*Muta
 	settings := MutableBucketSettings{}
 
 	settings.FlushEnabled = data.Controllers.Flush != ""
-	settings.ReplicaIndexDisabled = !data.ReplicaIndex
+	replicaIndexDisabled := !data.ReplicaIndex
+	settings.ReplicaIndexDisabled = &replicaIndexDisabled
```

Comment:
So this is the JSON from the server?  Can it not contain a ReplicaIndex field, and what happens in that case?  I'd expect the final Protostellar result to not have that field set.  But hmm - just saw that ListBucketsResponse has most fields non-optional.  Is that what we want?

Reply:
replicaIndex is not present for ephemeral buckets. The only other field I can see not always being present is flush. But flush is either set to a uri or not set, so we'd have it as always true or nil - which is a bit odd IMO.

I think it could be argued that ephemeral not having it set is the same as false too. I think this change is actually wrong and we should just ignore the field on the create side for ephemeral buckets.